persistence/rest/src/main/java/org/infinispan/persistence/rest/RestStore.java

```diff
@@ -58,18 +56,20 @@
 import io.netty.handler.codec.http.HttpResponse;
 import io.netty.handler.codec.http.HttpResponseStatus;
 import io.netty.handler.codec.http.HttpVersion;
+import io.reactivex.Flowable;
 import net.jcip.annotations.ThreadSafe;
 
 /**
  * RestStore.
  *
  * @author Tristan Tarrant
  * @since 6.0
+ * @deprecated This cache store will be changed to only implement {@link org.infinispan.persistence.spi.CacheLoader}
```

Comment:
Aren't we using it for rolling upgrades?

Reply:
Nope, Hot Rod only